BatteryAbnormalConditions.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Diagnostics;
+
+namespace BatteryCharging
+{
+    public class BatteryAbnormalConditions
+    {
+        static readonly string LowBreach = "Breaching Point is Low";
+        static readonly string HighBreach = "Breaching Point is High";
+        static bool FaultyBattery = false;
+        public static void CheckBreachingPoint(string factorName, float factorValue, float minimumFactorLimit, float maximumFactorLimit)
```

Comment:
CheckBreachingPoint is open for modification , if new Battery parameters are added in the feature. refactor the function parameter list

Reply:
Could you please elaborate the changes and guide me for a solution ?